executor/newdeploy/newdeploymgr.go

```diff
@@ -547,19 +548,19 @@ func (deploy *NewDeploy) fnDelete(fn *crd.Function) (*fscache.FuncSvc, error) {
 }
 
 func (deploy *NewDeploy) getObjName(fn *crd.Function) string {
-	return fmt.Sprintf("%v-%v",
-		fn.Metadata.Name,
-		deploy.instanceID)
+	return strings.ToLower(fmt.Sprintf("newdeploy-%v-%v-%v", fn.Metadata.Name, fn.Metadata.Namespace, deploy.instanceID))
```

Comment:
I'm just nitpicking, but an object "c" in namespace "a-b" will have the same name as an object "b-c" in namespace "a".  You could solve it by adding a small random string to the name.